cylc/flow/scripts/validate_install_play.py

```diff
@@ -101,7 +102,11 @@ def main(parser: COP, options: 'Values', workflow_id: Optional[str] = None):
     orig_source = workflow_id
     source = get_source_location(workflow_id)
     log_subcommand('validate', source)
-    asyncio.run(cylc_validate(parser, options, str(source)))
+
+    # Copy options in case validate decides to store information
+    # in the options object:
+    val_opts = copy(options)
+    asyncio.run(cylc_validate(parser, val_opts, str(source)))
```

Comment:
I was kinda relying on the old behaviour for the rose-stem plugin work (because we need to avoid re-running the plugin for the combined commands).

Did you get to the bottom of why `cylc validate` is modifying the options in the first place?

Reply:
No, but I imagine it's just a place to store data - since you need it for rose stem I'll investigate.